resources/views/components/accordion.blade.php

```diff
@@ -4,7 +4,7 @@
 <ul class="accordion">
     @foreach($items as $key=>$item)
         <li>
-            <a href="#definition-{{ $key }}">{{ $item['title'] }}</a>
+            <a href="#definition-{{ $key }}"><span aria-hidden="true"></span>{{ $item['title'] }}</a>
```

Comment:
Does this validate with this empty tag? Or are you filling it with something after load?

Reply:
@nickdenardis it is filled with a pseudo-element from CSS for ::content + or - so not sure if that counts? I could put something in there if we really need to, but I didn't want to add it since the content comes from CSS.

Are you talking about only HTML validation?

Reply:
Ok. We'll see how this all shakes out when running the tests on dev next week :)